Doc/library/math.rst

```diff
@@ -607,7 +607,7 @@ Summation and product functions
 
    Roughly equivalent to::
 
-       sum(map(operator.mul, p, q, strict=True))
+       sum(px * qx for px, qx in zip(p, q, strict=True))
```

Comment:
I would suggest leaving as is but adding a sentence below such as:
```
sum(map(operator.mul, p, q, strict=True))

where the multiply operator is used to multiply each item in the *p* and *q* iterables together 
and sum the products.
```

I'm not sure but I suspect the original command is more performant. I'm fine also with adding the proposed item as an alternative illustration.

I would leave the `.c` file unchanged using map and operator.mul.

Reply:
ISTM the proposed extra sentence just repeats the first sentence before the rough equivalent.